editor/import/resource_importer_scene.cpp

```diff
@@ -980,6 +1017,65 @@ bool ResourceImporterScene::get_internal_option_visibility(InternalImportCategor
 		case INTERNAL_IMPORT_CATEGORY_NODE: {
 		} break;
 		case INTERNAL_IMPORT_CATEGORY_MESH_3D_NODE: {
+			const bool generate_physics =
+					p_options.has("generate/physics") &&
```

Comment:
If I am not mistaken, using `.has()` is not needed, it is warranted that you will always get all the properties filled in (and if not there is likely a bug).

Reply:
Calling `.has()` is necessary, to avoid crash the engine while using the `constant` version of `operator[]`: https://github.com/godotengine/godot/blob/master/core/templates/map.h#L662